### ISSUE TITLE
How can i get the avatar?

### ISSUE THREAD

**Issue by Classifiee** [justacoincidence/CharliesCrashers#1]
I wanna get an avatar from the list, but i dont know how to download it and put it into unity.

**Comment by randy555** [comment]
https://docs.vrchat.com/docs/creating-your-first-avatar
https://www.youtube.com/watch?v=7sqnAbPxYnQ

The YouTube link is a walkthrough and the docs link is the link to all the documentation you would need to upload the avatar. I haven't uploaded an avatar in vrchat since 2019 so some things have changed. That video and documentation should help you.

**Comment by Classifiee** [comment]
> https://docs.vrchat.com/docs/creating-your-first-avatar https://www.youtube.com/watch?v=7sqnAbPxYnQ
> 
> The YouTube link is a walkthrough and the docs link is the link to all the documentation you would need to upload the avatar. I haven't uploaded an avatar in vrchat since 2019 so some things have changed. That video and documentation should help you.

Hey, sorry for the confusion i know how to upload avatars from unity to vrchat, but the avatars from the avatar list provided when downloaded give me only the .vrca file. When i put it through a converter to actually get something it gives me stuff that presumedly should be in a asset folder but no model or sometimes armature. There is no actual asset folder either. It's just weird ya know?

**Comment by randy555** [comment]
> > https://docs.vrchat.com/docs/creating-your-first-avatar https://www.youtube.com/watch?v=7sqnAbPxYnQ
> > The YouTube link is a walkthrough and the docs link is the link to all the documentation you would need to upload the avatar. I haven't uploaded an avatar in vrchat since 2019 so some things have changed. That video and documentation should help you.
> 
> Hey, sorry for the confusion i know how to upload avatars from unity to vrchat, but the avatars from the avatar list provided when downloaded give me only the .vrca file. When i put it through a converter to actually get something it gives me stuff that presumedly can make but no model or sometimes armature.

Fair enough, I haven't actually tried uploading these avatars let alone messing around with unity for years. Figured I'd try and help but my knowledge ends here. I thought it was weird for the files to be listed as vrca files. Theres gotta be some way to do it, at least I'd think. Not sure though like I said, haven't played around with this stuff for years so I've forgotten most everything. Consider making an account on UnknownCheats.me and posting a question there. Those people have quite a bit of knowledge.

**Comment by Classifiee** [comment]
> > > https://docs.vrchat.com/docs/creating-your-first-avatar https://www.youtube.com/watch?v=7sqnAbPxYnQ
> > > The YouTube link is a walkthrough and the docs link is the link to all the documentation you would need to upload the avatar. I haven't uploaded an avatar in vrchat since 2019 so some things have changed. That video and documentation should help you.
> > 
> > 
> > Hey, sorry for the confusion i know how to upload avatars from unity to vrchat, but the avatars from the avatar list provided when downloaded give me only the .vrca file. When i put it through a converter to actually get something it gives me stuff that presumedly can make but no model or sometimes armature.
> 
> Fair enough, I haven't actually tried uploading these avatars let alone messing around with unity for years. Figured I'd try and help but my knowledge ends here. I thought it was weird for the files to be listed as vrca files. Theres gotta be some way to do it, at least I'd think. Not sure though like I said, haven't played around with this stuff for years so I've forgotten most everything. Consider making an account on UnknownCheats.me and posting a question there. Those people have quite a bit of knowledge.

Thanks for the recommendation i 100% did not know of that website. I'll go ahead and try the website.

**Comment by Mrgoldilocks** [comment]
btw if that didnt work there is a YT tutorial on ripping that has instructions for VCRA files

https://youtu.be/_XOOsh09IqU
